### PR TITLE
A11y microsite

### PR DESCRIPTION
**Pull request by LDubya**
Placeholder for a11y microsite pull request. Description of modifications and new functionality can go here.

### CODE REVIEW

File: _teams/accessibility/research-in-biomedical-accessibility.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+# make sure to add page to _data/teams.yml
+
+name: Research in Biomedical Accessibility
```
Comment (sehilyi) [inline _teams/accessibility/research-in-biomedical-accessibility.md:4]: To make it consistent with our paper (i.e., biomedical → life sciences).

```suggestion
name: Accessibility in Life Sciences Data Resources
```

File: _teams/accessibility/research-in-biomedical-accessibility.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+image: a11y-results.png
+image_alt: A figure containing three graphs. Graph A shows that 96% of data portals have issues. Graph B shows the percentage of those issues by accessibility categories, with contrast being the most prevalent issue, and label_missing, alt_missing, and label_empty highlighted. Graph C is a pie chart showing that 78.8% of alt text in journals are entirely missing, while 21.3% are uninformative.
+
+publications:
+  - lyi-2024-F8FYKQP6
+
+---
+
+## Evaluation of Accessibility in Biomedical Resources
```
Comment (sehilyi) [inline _teams/accessibility/research-in-biomedical-accessibility.md:16]:
```suggestion
## Evaluation of Accessibility in Life Sciences Data Resources
```

File: _teams/accessibility/research-in-biomedical-accessibility.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Current Accessibility Status
+Our evaluation revealed that nearly all of the assessed websites (96%) had severe accessibility issues. The most common issue identified was the absence of crucial metadata, such as alternative text for images, which creates significant barriers for users with blindness and low vision (BLV). Without such metadata, it is impossible for BLV users to fully understand the content displayed in biomedical data visualizations (Refer to figure on the top).
+
+## Research Aims
+Our primary goal is to improve the accessibility of biomedical resources by identifying the key barriers and developing strategies to address them. Through this work, we aim to ensure that biomedical websites and tools comply with the Web Content Accessibility Guidelines (WCAG) and are accessible to people with disabilities.
+
+## Our Contributions
+- **Evaluation Results**: We identified a range of accessibility failures, with missing alt texts being one of the most prevalent issues.
+- **Tools Used**: We employed computational accessibility tools such as WAVE and Axe to conduct our analysis.
```
Comment (sehilyi) [inline _teams/accessibility/research-in-biomedical-accessibility.md:28]: A bit of revision to highlight the scientific contribution better

```suggestion
- **Mixed-methods Approach**: We employed computational accessibility tools, such as WAVE and Axe, to conduct our analysis at scale, in addition to conducting user evaluation for selected resources.
```

File: _teams/accessibility/research-in-biomedical-accessibility.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Our lab been conducting extensive research evaluating the accessibility of biomedical websites and resources. In this ongoing project, we assessed over 3,000 biomedical data portals and journal websites using accessibility evaluation tools like [WAVE](https://wave.webaim.org) and [Axe](https://www.deque.com/axe/).
+
+## Current Accessibility Status
+Our evaluation revealed that nearly all of the assessed websites (96%) had severe accessibility issues. The most common issue identified was the absence of crucial metadata, such as alternative text for images, which creates significant barriers for users with blindness and low vision (BLV). Without such metadata, it is impossible for BLV users to fully understand the content displayed in biomedical data visualizations (Refer to figure on the top).
+
+## Research Aims
+Our primary goal is to improve the accessibility of biomedical resources by identifying the key barriers and developing strategies to address them. Through this work, we aim to ensure that biomedical websites and tools comply with the Web Content Accessibility Guidelines (WCAG) and are accessible to people with disabilities.
```
Comment (sehilyi) [inline _teams/accessibility/research-in-biomedical-accessibility.md:24]:
```suggestion
Our primary goal is to improve the accessibility of life sciences resources by identifying the key barriers and developing strategies to address them. Through this work, we aim to ensure that biomedical websites and tools comply with the Web Content Accessibility Guidelines (WCAG) and are accessible to people with disabilities.
```

File: _teams/accessibility/research-in-biomedical-accessibility.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  - lyi-2024-F8FYKQP6
+
+---
+
+## Evaluation of Accessibility in Biomedical Resources
+
+Our lab been conducting extensive research evaluating the accessibility of biomedical websites and resources. In this ongoing project, we assessed over 3,000 biomedical data portals and journal websites using accessibility evaluation tools like [WAVE](https://wave.webaim.org) and [Axe](https://www.deque.com/axe/).
+
+## Current Accessibility Status
+Our evaluation revealed that nearly all of the assessed websites (96%) had severe accessibility issues. The most common issue identified was the absence of crucial metadata, such as alternative text for images, which creates significant barriers for users with blindness and low vision (BLV). Without such metadata, it is impossible for BLV users to fully understand the content displayed in biomedical data visualizations (Refer to figure on the top).
```
Comment (sehilyi) [inline _teams/accessibility/research-in-biomedical-accessibility.md:21]:
```suggestion
Our evaluation revealed that nearly all of the assessed websites (96%) had severe accessibility issues. The most common issue identified was the absence of crucial metadata, such as alternative text for images, which creates significant barriers for users with blindness and low vision (BLV). Without such metadata, it is impossible for BLV users to fully understand the content displayed in life sciences data visualizations (Refer to the figure on the top).
```

File: _teams/accessibility/research-in-biomedical-accessibility.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## Research Aims
+Our primary goal is to improve the accessibility of biomedical resources by identifying the key barriers and developing strategies to address them. Through this work, we aim to ensure that biomedical websites and tools comply with the Web Content Accessibility Guidelines (WCAG) and are accessible to people with disabilities.
+
+## Our Contributions
+- **Evaluation Results**: We identified a range of accessibility failures, with missing alt texts being one of the most prevalent issues.
+- **Tools Used**: We employed computational accessibility tools such as WAVE and Axe to conduct our analysis.
+- **Addressing Gaps**: Our findings highlight the need for improving accessibility across biomedical websites, particularly in providing alternative texts and complying with WCAG.
```
Comment (sehilyi) [inline _teams/accessibility/research-in-biomedical-accessibility.md:29]:
```suggestion
- **Addressing Gaps**: Our findings highlight the need for improving accessibility across life sciences websites, particularly in providing alternative texts and complying with WCAG.
```

File: _teams/accessibility/research-in-biomedical-accessibility.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+## Research Aims
+Our primary goal is to improve the accessibility of life sciences resources by identifying the key barriers and developing strategies to address them. Through this work, we aim to ensure that biomedical websites and tools comply with the Web Content Accessibility Guidelines (WCAG) and are accessible to people with disabilities.
+
+## Our Contributions
+- **Evaluation Results**: We identified a range of accessibility failures, with missing alt texts being one of the most prevalent issues.
+- **Mixed-methods Approach**: We employed computational accessibility tools, such as WAVE and Axe, to conduct our analysis at scale, in addition to conducting user evaluation for selected resources.
+- **Addressing Gaps**: Our findings highlight the need for improving accessibility across life sciences websites, particularly in providing alternative texts and complying with WCAG.
+
+## Call to Action
+We invite you to participate in this effort. We are looking for feedback and suggestions from the broader scientific and accessibility communities and potential collaborations for academic and non-academic activities.
```
Comment (sehilyi) [inline _teams/accessibility/research-in-biomedical-accessibility.md:32]:
```suggestion
---
# make sure to add page to _data/teams.yml

name: Accessibility in Life Sciences Data Resources
team_id: accessibility
active: true

image: a11y-results.png
image_alt: A figure containing three graphs. Graph A shows that 96% of data portals have issues. Graph B shows the percentage of those issues by accessibility categories, with contrast being the most prevalent issue, and label_missing, alt_missing, and label_empty highlighted. Graph C is a pie chart showing that 78.8% of alt text in journals are entirely missing, while 21.3% are uninformative.

publications:
  - lyi-2024-F8FYKQP6

---

## Evaluation of Accessibility in Life Sciences Data Resources

Our lab has been conducting extensive research evaluating the accessibility of life sciences websites and data resources. In this ongoing project, we assessed over 3,000 data portals and journal websites using accessibility evaluation tools, like [WAVE](https://wave.webaim.org) and [Axe](https://www.deque.com/axe/). We also evaluated selected data portals with an actual screen reader users to identify real-world barriers for performing important tasks (e.g., data discovery).

## Current Accessibility Status
Our evaluation revealed that nearly all assessed websites (96%) had severe accessibility issues. The most common issue identified was the absence of crucial metadata, such as alternative text for images, which creates significant barriers for users with blindness and low vision (BLV). Without such metadata, it is impossible for BLV users to fully understand the content displayed in life sciences data visualizations (Refer to the figure on the top).

## Research Aims
Our primary goal is to improve the accessibility of life sciences resources by identifying the key barriers and developing strategies to address them. Through this work, we aim to ensure that life sciences websites and tools comply with the Web Content Accessibility Guidelines (WCAG) and are accessible to people with disabilities.

## Our Contributions
- **Evaluation Results**: We identified a range of accessibility failures, with missing alt texts being one of the most prevalent issues.
- **Mixed-methods Approach**: We employed computational accessibility tools to conduct our analysis at scale, in addition to conducting user evaluation for selected resources.
- **Addressing Gaps**: Our findings highlight the need for improving accessibility across life sciences websites, particularly in providing alternative texts and complying with WCAG.

## Call to Action
We invite you to participate in this effort. We seek feedback and suggestions from the broader scientific and accessibility communities and potential collaborations for academic and non-academic activities.
```

File: _teams/accessibility/our-accessibility-journey.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+name: Our Accessibility Journey
+team_id: accessibility
+active: true
+---
+
+<!-- Every web interface should be accessible, but inaccessible web experiences are the norm. -->
+
+When the HIDIVE lab began our accessibility research, we first focused on improving screen reader accessibility for data visualizations. 
+Recognizing that grammar-based visualization tools like the Gosling toolkit could be converted into meaningful text descriptions, {% include get-name-degree.html title="Thomas Smits" %} led the AltGosling project to automate description generation for genome-mapped data visualizations.
+
```
Comment (thomcsmits) [inline _teams/accessibility/our-accessibility-journey.md:13]: I don't really see the added value of adding degree titles to the text? It also makes it a bit harder to read. If someone is interested in degrees, they can always check out the individual pages. Perhaps we can even hyperlink these directly from this text?

**Review by thomcsmits — COMMENTED** [review]
Added some changes here as well: https://github.com/LDubya/gehlenborglab-website/pull/16

File: _teams/accessibility/our-accessibility-journey.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+{% include get-name-degree.html title="Sofia Rojas" %} and {% include get-name-degree.html title="Andrew Mar" %} joined the lab as Research Assistants, contributing to several of the team's projects. 
+{% include get-name-degree.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 
+
+We continue to explore ways to make biomedical data visualizations accessible for people with a wideer range of disabilities. 
+The preprint *Using OpenKeyNav to Enhance the Keyboard-Accessibility of Web-based Data Visualization Tools*, based on an open-source software library by {% include get-name-degree.html title="Lawrence Weru" %}, explores how to improve the keyboard accessibility of web-based data visualization tools for people with motor disabilities.
+The preprint was accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
```
Comment (thomcsmits) [inline _teams/accessibility/our-accessibility-journey.md:24]:
```suggestion
The preprint was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 

We also looked into mental models of visualizations related to individuals who are blind, led by by {% include get-name-degree.html title="Thomas Smits" %}. The preprint for this was also accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
```

**Review by thomcsmits — CHANGES_REQUESTED** [review]

File: _teams/accessibility/our-accessibility-journey.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+{% include get-name-degree.html title="Sofia Rojas" %} and {% include get-name-degree.html title="Andrew Mar" %} joined the lab as Research Assistants, contributing to several of the team's projects. 
+{% include get-name-degree.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 
+
+We continue to explore ways to make biomedical data visualizations accessible for people with a wideer range of disabilities. 
+The preprint *Using OpenKeyNav to Enhance the Keyboard-Accessibility of Web-based Data Visualization Tools*, based on an open-source software library by {% include get-name-degree.html title="Lawrence Weru" %}, explores how to improve the keyboard accessibility of web-based data visualization tools for people with motor disabilities.
+The preprint was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 
+
```
Comment (sehilyi) [inline _teams/accessibility/our-accessibility-journey.md:25]: Sehi to add a paragraph about the K

File: _teams/accessibility/our-accessibility-journey.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+Recognizing the under-representation of people with disabilities in the biomedical sciences,
+we began quantifying the state accessibility of research journals and data portals, crucial for biomedical research, in a project led by {% include get-member-url.html title="Sehi L'Yi" %}.
+
+We began making remediations to our website to better conform to WCAG 2.1 level AA. This included adding alt text to over 100 images and ensuring accessible navigation throughout the site.
+
+{% include get-member-url.html title="Sofia Rojas" %} and {% include get-member-url.html title="Andrew Mar" %} joined the lab as Research Assistants, contributing to several of the team's projects. 
+{% include get-member-url.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
```
Comment (thomcsmits) [inline _teams/accessibility/our-accessibility-journey.md:21]:
```suggestion
{% include get-member-url.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 

We also looked into mental models of visualizations related to individuals who are blind, led by by {% include get-name-degree.html title="Thomas Smits" %}. The preprint for this was also accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
```

File: _teams/accessibility/our-accessibility-journey.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+<!-- Every web interface should be accessible, but inaccessible web experiences are the norm. -->
+
+<!-- When the HIDIVE lab began our accessibility research, we first focused on improving screen reader accessibility for data visualizations.  -->
+The HIDIVE Lab accessibility research, led by {% include get-member-url.html title="Sehi L'Yi" %}, initially focused on improving screen reader accessibility for data visualizations. 
+Recognizing that grammar-based visualization tools like the Gosling toolkit could be converted into meaningful text descriptions, {% include get-member-url.html title="Thomas Smits" %} led the AltGosling project to automate description generation for genome-mapped data visualizations.
+
+Recognizing the under-representation of people with disabilities in the biomedical sciences,
+we began quantifying the state accessibility of research journals and data portals, crucial for biomedical research, in a project led by {% include get-member-url.html title="Sehi L'Yi" %}.
```
Comment (thomcsmits) [inline _teams/accessibility/our-accessibility-journey.md:16]:
```suggestion
we began quantifying the state accessibility of research journals and data portals, crucial for biomedical research, in a project led by Sehi L'Yi.
```

File: _teams/accessibility/our-accessibility-journey.md [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+
+We began making remediations to our website to better conform to WCAG 2.1 level AA. This included adding alt text to over 100 images and ensuring accessible navigation throughout the site.
+
+{% include get-member-url.html title="Sofia Rojas" %} and {% include get-member-url.html title="Andrew Mar" %} joined the lab as Research Assistants, contributing to several of the team's projects. 
+{% include get-member-url.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
```
Comment (thomcsmits) [inline _teams/accessibility/our-accessibility-journey.md:21]:
```suggestion
Andrew Mar's contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
```

File: _teams/accessibility/our-accessibility-journey.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+we began quantifying the state accessibility of research journals and data portals, crucial for biomedical research, in a project led by {% include get-member-url.html title="Sehi L'Yi" %}.
+
+We began making remediations to our website to better conform to WCAG 2.1 level AA. This included adding alt text to over 100 images and ensuring accessible navigation throughout the site.
+
+{% include get-member-url.html title="Sofia Rojas" %} and {% include get-member-url.html title="Andrew Mar" %} joined the lab as Research Assistants, contributing to several of the team's projects. 
+{% include get-member-url.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 
+
+We also looked into mental models of visualizations related to individuals who are blind, led by by {% include get-name-degree.html title="Thomas Smits" %}. The preprint for this was also accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
+
```
Comment (sehilyi) [inline _teams/accessibility/our-accessibility-journey.md:24]:
```suggestion

Our accessibility efforts led to more opportunities to expand our work. {% include get-member-url.html title="Sehi L'Yi" %} has recently received the NIH K99/R00 Pathway to Independence Award to continue work on accessibility. This project proposes to increase the diversity of the genomics workforce through accessible data and visualization. This includes conducting comprehensive evaluations of existing biomedical data resources and building tools and frameworks that can increase the accessibility of biomedical research and education. This grant will help {% include get-member-url.html title="Sehi L'Yi" %} to build an independent research group to work on this area in the future.

```

File: _teams/accessibility/our-accessibility-journey.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+We continue to explore ways to make biomedical data visualizations accessible for people with a wideer range of disabilities. 
+The preprint *Using OpenKeyNav to Enhance the Keyboard-Accessibility of Web-based Data Visualization Tools*, based on an open-source software library by {% include get-member-url.html title="Lawrence Weru" %}, explores how to improve the keyboard accessibility of web-based data visualization tools for people with motor disabilities.
+The preprint was accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
+
```
Comment (sehilyi) [inline _teams/accessibility/our-accessibility-journey.md:28]:
```suggestion

Our accessibility efforts led to more opportunities to expand our work. {% include get-member-url.html title="Sehi L'Yi" %} has recently received the NIH K99/R00 Pathway to Independence Award to continue work on accessibility. This project proposes to increase the diversity of the genomics workforce through accessible data and visualization. This includes conducting comprehensive evaluations of existing biomedical data resources and building tools and frameworks that can increase the accessibility of biomedical research and education. This grant will help {% include get-member-url.html title="Sehi L'Yi" %} to build an independent research group to work on this area in the future.

```

File: _teams/accessibility/our-accessibility-journey.md [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+name: Our Accessibility Journey
+team_id: accessibility
+active: true
+---
+
+<!-- Every web interface should be accessible, but inaccessible web experiences are the norm. -->
+
+<!-- When the HIDIVE lab began our accessibility research, we first focused on improving screen reader accessibility for data visualizations.  -->
+The HIDIVE Lab accessibility research, led by {% include get-member-url.html title="Sehi L'Yi" %}, initially focused on improving screen reader accessibility for data visualizations.
```
Comment (thomcsmits) [inline _teams/accessibility/our-accessibility-journey.md:12]:
```suggestion
The HIDIVE Lab accessibility research, led by {% include get-member-url.html title="Sehi L'Yi" %}, initially focused on improving accessibility for genome-mapped data visualizations. 
```

File: _teams/accessibility/our-accessibility-journey.md [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+name: Our Accessibility Journey
+team_id: accessibility
+active: true
+---
+
+<!-- Every web interface should be accessible, but inaccessible web experiences are the norm. -->
+
+<!-- When the HIDIVE lab began our accessibility research, we first focused on improving screen reader accessibility for data visualizations.  -->
+The HIDIVE Lab accessibility research, led by {% include get-member-url.html title="Sehi L'Yi" %}, initially focused on improving screen reader accessibility for data visualizations. 
+Recognizing that grammar-based visualization tools like the Gosling toolkit could be converted into meaningful text descriptions, {% include get-member-url.html title="Thomas Smits" %} led the AltGosling project to automate description generation for genome-mapped data visualizations.
```
Comment (thomcsmits) [inline _teams/accessibility/our-accessibility-journey.md:13]:
```suggestion
Recognizing that grammar-based visualization tools like the Gosling toolkit could be converted into meaningful text descriptions, {% include get-member-url.html title="Thomas Smits" %} led the AltGosling project to automate description generation for genomics data visualizations.
```

File: teams.md [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+<table class="projects">
+
+{% for team in site.teams %}
+{% if team.active and team.team_home %}
+<tr>
+<td>
+{% if team.gallery.size > 0 %}
+<a href="{{team.url}}"><img class="thumb"
+            src="/assets/img/publications/thumbnail/{{ team.gallery.first[0] }}"
+            alt="{{ team.gallery.first[1] }}"></a>
```
Comment (thomcsmits) [inline teams.md:34]: @LDubya am I correct to see here that the image and alt don't match indices?
```suggestion
<a href="{{team.url}}"><img class="thumb"
            src="/assets/img/publications/thumbnail/{{ team.gallery.first[0] }}"
            alt="{{ team.gallery.first[0] }}"></a>
```

Comment (LDubya) [reply teams.md:34]: The indices follow the gallery format, so `0` index is the filename and `1` index is the description

```
gallery:
  cistrome-explorer.png: 'Cistrome Explorer provides scalable, interactive, and modular visualizations for epigenomics data with associated metadata.'
  cistrome-explorer-toolkit.png: 'Cistrome Explorer integrated Cistrome DB Toolkit that contains huge collection of public ChIP-seq, DNase-seq, and ATAC-seq datasets.'
```

**Review by thomcsmits — APPROVED** [review]
LGTM!

**Pull request merged**